generators/bdk/java/templates/ext-app/application.yaml.ejs

```diff
@@ -0,0 +1,49 @@
+# BDK Core Configuration
+bdk:
+  host: <%= host %>
+  bot:
+    username: <%= username %>
+    privateKey:
+      path: <%= privateKeyPath %>
+
+  app:
+    appId: <%= appId %>
+    privateKey:
+      path: <%= privateKeyPath %>
```

Comment:
I was wondering if we should consider having a separate key for the app/bot
I'm not sure it is an issue, what do you think @symphony-thibault?

Reply:
I don't think we need it because we have to generate the key pair twice and most of the time I prefer to replace them with my own keys.